packages/design-system-docs/src/pages/utilities/display-visibility/_display-visibility.docs.scss

```diff
@@ -11,9 +11,9 @@ Display
 
 > The display CSS property specifies the type of rendering box used for an element. In HTML, default display property values are taken from behaviors described in the HTML specifications or from the browser/user default stylesheet. &mdash; [MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/display)
 
-**Format**: `.ds-u-display--[block|inline-block|flex|none]`
+**Format**: `.ds-u-display--[inline|block|inline-block|flex|none]`
 
-**Responsive format**: `.ds-u-[breakpoint]-display--[block|inline-block|flex|none]`
```

Comment:
Could you move the inline after block? So the ordering is `block`, `inline`, `inline-block` etc

Reply:
I think `inline` should follow `block` so you can actually see the inline elements behaving together. I think its ok if it looks wonkier when you resize the screen, as thats expected with the inline element. 
<img width="766" alt="Screen Shot 2020-06-04 at 2 16 52 PM" src="https://user-images.githubusercontent.com/5424713/83801191-56237880-a66e-11ea-9fdb-8f50a9b300bd.png">

Reply:
`block` also needs to go first in my opinon. If you look at [USWDS](https://designsystem.digital.gov/utilities/display/) they order it with block first. In fact, I think we should would move flex to second as well.

Reply:
Okay, let's take a look at the change to this order - block|flex|inline|inline-block.